docs/source-stepping.md

```diff
@@ -5,11 +5,32 @@ but occasionally, it may help to see what FakeItEasy is doing under the hood.
 FakeItEasy uses [SourceLink](https://github.com/dotnet/sourcelink) to make it
 a breeze to step into the FakeItEasy source code.
 
-To enable SourceLink to work in Visual Studio:

```

Comment:
"To enable SourceLink to work," sounds funny to me. Actually on reread, so does "To enable SourceLink to work in Visual Studio:", but slightly less so. I think you could drop "to work".

Whatever change you decided to make (if any!) feel free to amend.

Reply:
Thanks. Amended!